src/hotspot/share/cds/classListParser.cpp

```diff
@@ -466,16 +473,16 @@ InstanceKlass* ClassListParser::load_class_from_source(Symbol* class_name, TRAPS
           _interfaces->length(), k->local_interfaces()->length());
   }
 
+  // This tells JVM_FindLoadedClass to not find this class.
```

Comment:
This comment can be deleted.

The class `k` used to be defined by the boot loader. As a result, when `JVM_FindLoadedClass` looked up a class in the boot/platform/app loaders, it may inadvertently find `k`.

However, after this PR, `k` is no longer defined by the boot loader.

Reply:
Removed the comment.